bigtable/tests/admin_integration_test.cc

```diff
@@ -12,185 +12,269 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <gmock/gmock.h>
+
+#include <absl/memory/memory.h>
 #include <absl/strings/str_join.h>
 #include <google/protobuf/text_format.h>
 #include <google/protobuf/util/message_differencer.h>
-#include <sstream>
-#include "bigtable/admin/admin_client.h"
-#include "bigtable/admin/table_admin.h"
 
-namespace {
+#include <bigtable/admin/admin_client.h>
+#include <bigtable/admin/table_admin.h>
+#include "bigtable/client/testing/table_integration_test.h"
+
+#include <string>
+#include <vector>
+
+using namespace std;
 namespace admin_proto = ::google::bigtable::admin::v2;
 
-void CheckInstanceIsEmpty(bigtable::TableAdmin& admin) {
-  std::vector<std::string> table_names;
-  auto table_list = admin.ListTables(admin_proto::Table::NAME_ONLY);
-  if (not table_list.empty()) {
-    throw std::runtime_error("Expected empty instance in integration test");
-  }
-  std::cout << "Initial ListTables() successful" << std::endl;
-}
+namespace bigtable {
```

Comment:
If these types and functions are purely local then they should be in the anonymous namespace.